### DISCUSSION TITLE
How to spell stock symbols, like "AAPL", "AMZN", "GOOG"

### DISCUSSION THREAD

**Discussion by pmuvb** [espeak-ng/espeak-ng#2042] [General]
Hi,

How to make it to spell stock symbols instead of it reading them as whole words?

I also want it to avoid changing an intonation at the end of the word, '-z' doesn't work.

I can make a call for each letter but I want to avoid that.

Thanks!

**Comment by fabiolimace** [comment]
A definitive way of doing it is by adding lines to `dictsource/en_list`, after cloning the repository, and compiling the language again. For example, the line below tells espeak-ng to spell GOOG, letter by letter, as an abbreviation.

```
goog    $abbrev
```

After changing `dictsource/en_list`, run this command inside of `dictsource` directory:

```
sudo espeak-ng --path=/usr/share/espeak-ng-data/ --compile=en
```

After compilation, GOOG will be always heard as individual letters.
